util/src/Pos/Util/Log/LoggerConfig.hs

```diff
@@ -274,6 +275,25 @@ defaultInteractiveConfiguration minSeverity =
     in
     LoggerConfig{..}
 
+-- | Handler to override another console 'LogHandler' and disable logging
+-- to console altogether.
+disabledConsoleConfiguration :: LoggerConfig
```

Comment:
we will write a test to demonstrate this..

Reply:
:pray: